### DISCUSSION TITLE
Welcome to AI discussions at OWASP!

### DISCUSSION THREAD

**Discussion by robvanderveer** [OWASP/www-project-ai-security-and-privacy-guide#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by nikhilsingh38** [comment]
Can anyone help me with the assessment for AI?

**Reply by robvanderveer** [reply]
What would you like to assess?
-Accuracy
-Security
-Sensitivity to adversarial attack (part of security actually)?
-Unwanted bias
-Purpose-binding data usage / consent
-Legal/ethical purpose
-Maintainability
-Maturity of the development process
-Transparency
-Explainability
-Performance
-Scalability
-Reliability
-Privacy rights
?

**Comment by nikhilsingh38** [comment]
Hi Rob,

Thanks for your email. Certainly! I’m interested in assessing all of these
parameters.

Thanks & best Regards,

**Comment by stephanielaforge** [comment]
I have a recommendation:  Any time an acronym is used for the first time in a page, that acronym should be spelled out in parentheses. Use of common acronyms may seem obvious to the person writing the article or page, but are not always known or immediately obvious to readers coming to learn.  The typical convention for this is, for example: Whole Phrase Spelled Out (WPSO). 
And once done, you can easily refer to the same acronym without having to spell it out each time. 

You don't always have to do this. You have a list of privacy frameworks, for example... but they are grouped and referred to as privacy frameworks. So, while they are not spelled out, they are defined. 

MFA Flow:
KYC Requirements:
ML: 
EDPS:
LIME:
SHAP:
T&S:
